pkg/apis/v1alpha2/aksnodeclass_hash_test.go

```diff
@@ -103,4 +104,18 @@ var _ = Describe("Hash", func() {
 		}
 		Expect(nodeClass.Hash()).To(Equal(otherNodeClass.Hash()))
 	})
+	// This test is a sanity check to update the hashing version if the algorithm has been updated.
+	// Note: this will only catch a missing version update, if the staticHash hasn't been updated yet.
+	It("when hashing algorithm updates, we should update the hash version", func() {
+		// If you update the staticHash value, you need to update v1alpha2.AKSNodeClassHashVersion, and set currentHashVersion to the same value
```

Comment:
Wondering if this comment makes more sense next to `staticHash`, or rephrased

Reply:
Updated there, removed here